mod/b_extended_profile_collab/views/default/profile/tab-content.php

```diff
@@ -122,7 +132,15 @@
             }
         }
 
-        if(!$content){
+        if($field == "event_calendar"){    
+            $events = event_calendar_get_personal_events_for_user(elgg_get_page_owner_guid(), 5);
+            if(!$events){
+                echo '<div class="mrgn-lft-sm mrgn-bttm-md">' . elgg_echo('event_calendar:no_events_found11') . '</div>';
```

Comment:
There is an extra 11 at the end of the no_events_found in both files. Everything else looks good though.